protos/feast/core/DataSource.proto

```diff
@@ -34,6 +34,7 @@ message DataSource {
     STREAM_KAFKA = 3;
     STREAM_KINESIS = 4;
     BATCH_REDSHIFT = 5;
+    BATCH_SQLSERVER = 6;
```

Comment:
Should we make this `MSSQLSERVER` (and elsewhere)? Their product is a bit poorly named :)